docs/source/general/extend_django_freeradius.rst

```diff
@@ -0,0 +1,73 @@
+======================================
+Customizing django-freeradius
```

Comment:
let's change the title to 'How to extend django-freeradius`

Reply:
It seems too long on navigation tab. :grimacing:

Reply:
Navigation tab? :smile:

Reply:
The left navigation menu. Nevermind, now I feel conveying information is more important than aesthetics. :sweat_smile:

Reply:
Try `Extending django-freeradius` then